README.md

```diff
@@ -67,8 +67,8 @@ with other versions of those libraries (set the `EXTERNAL_*libname*` option to
 * [protozero](https://github.com/mapbox/protozero) (>= 1.6.3)
 
 It also requires access to a database server running
-[PostgreSQL](https://www.postgresql.org/) 9.6+ and
```

Comment:
This is, what we test, so I'd leave it in. May be do something like (9.6+ works, 13+ strongly recommended)?

Reply:
fixed